pic-sure-api-war/src/main/java/edu/harvard/dbmi/avillach/LoggerReaderInterceptor.java

```diff
@@ -10,7 +10,7 @@
 import java.io.IOException;
 import java.io.InputStream;
 
-@Provider
+//@Provider
```

Comment:
Why was this changed? Are resource credentials still being redacted? We should never log or audit resource credentials.